Weather wear.js

```diff
@@ -0,0 +1,26 @@
+function todaysWeather(temperature){
+    if (temperature <= 0){
+        return "Wear a heavy winter jacket, hat, gloves, and scarf.";
+    }
+
+    if (temperature <= 5){
+        return "Wear a winter jacket, hat, and gloves.";
+    }
+
+    if (temperature <= 10){
+        return "Wear a winter jacket.";
+    }
+
+    if (temperature <= 15){
+        return "Wear a sweater shirt and a jean.";
+    }
+    if (temperature <= 20){
+        return "Wear a long sleeve shirt and a jean.";
+    }
+    else{
+        return "Wear a t-shirt and shorts.";
+    }
```

Comment:
I really like how you structured your function! There are many ways how functions like this can be written (for example, using `else if` or `switch/case`), but your version def has nice readability. This is a pattern also known as *early return*.

One tiny suggestion here is there is actually no need for an `else` case. So this is identical to the above:

```suggestion
    
    return "Wear a t-shirt and shorts.";
```

Again, it's about the code style and there is nothing wrong about your version, but when you have so-called default return, it's a little easier to add new cases (temp <= 30) and it improves readability a bit.